knx/dpt/types_registry.go

```diff
@@ -10,13 +10,29 @@ import (
 
 var (
 	types = [...]DatapointValue{
-		// 1.xxx
```

Comment:
I think that comment may stay. Unless you want to remove it for all sections.

Reply:
oups, that should not have been deleted!